Lesson21/index.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+alert('Enter two numbers to see if the first is greater than the second');
+
+let firstNumber = prompt('Please enter the first number', 0);
+let secondNumber = prompt('Please enter the second number', 0);
```

Comment:
It's better to convert prompt entered values into "number" type before converting.